app/views/directives/metrics.html

```diff
@@ -0,0 +1,91 @@
+<div class="metrics" ng-if="pod || deployment">
+  <div class="metrics-options">
+    <!-- Let the user select the container if we're showing pod metrics. -->
+    <div ng-if="pod.spec.containers.length" class="form-group">
+      <label for="selectContainer">Container:</label>
+      <div class="select-container">
+        <span ng-show="pod.spec.containers.length === 1">
+          {{pod.spec.containers[0].name}}
+        </span>
+        <select id="selectContainer"
+                ng-show="pod.spec.containers.length > 1"
+                ng-init="options.selectedContainer = pod.spec.containers[0]"
+                ng-model="options.selectedContainer"
+                ng-options="container.name for container in pod.spec.containers track by container.name">
+        </select>
+      </div>
+    </div>
+    <div class="form-group">
+      <label for="timeRange">Time Range:</label>
+      <select id="timeRange"
+              ng-model="options.timeRange"
+              ng-options="range.label for range in options.rangeOptions"
+              ng-disabled="metricsError">
+      </select>
+    </div>
+  </div>
+
+  <div ng-if="!loaded">Loading...</div>
+  <div ng-if="loaded && (usageByMetric | hashSize) === 0 && !metricsError">No metrics to display.</div>
+
+  <div ng-if="metricsError" class="empty-state-message text-center">
+    <h2>
+      <span class="pficon pficon-error-circle-o" aria-hidden="true"></span>
+      Metrics are not available.
+    </h2>
+
+    <p>An error occurred getting metrics for container {{options.selectedContainer.name}}<span ng-if="metricsURL">
+        from <a ng-href="{{metricsURL}}">{{metricsURL}}</a></span>.
+    </p>
+
+    <p ng-if="metricsError.status === 0 || metricsError.status === 404">
+      <div>If you have network connectivity, this could indicate a misconfiguration.</div>
+      <div>Please contact your system administrator.</div>
+    </p>
+
+    <p class="text-muted">
+      {{metricsError.details}}
+    </p>
+  </div>
+
+  <div ng-repeat="metric in metrics" ng-if="anyUsageByMetric(metric) && !metricsError">
+    <h3>
+      {{metric.label}}
+      <small ng-if="pod.spec.containers.length > 1">
+        <span ng-if="metric.containerMetric">Container Metrics</span>
```

Comment:
not a new issue, but is this actually right for network metrics, it looked like network metrics were at the pod level and not the container?